bokehjs/src/coffee/core/util/object.ts

```diff
@@ -27,6 +29,21 @@ export function clone<T>(obj: T): T {
   return extend({}, obj)
 }
 
+export function merge<T>(obj1: {[key: string] : Array<T>}, obj2: {[key: string]: Array<T>}): object {
```

Comment:
@clairetang6, I was wrong with the original signature. We define `merge()` for specific class of objects, so the definition here is correct. Though the return type could be more specific (the same as `obj1` and `obj2`).